src/main/java/com/mgmtp/radio/support/YouTubeHelper.java

```diff
@@ -0,0 +1,57 @@
+package com.mgmtp.radio.support;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.Video;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.mgmtp.radio.config.RadioConfig;
+import com.mgmtp.radio.config.YouTubeConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+@Component
+public class YouTubeHelper {
+    @Autowired
+    YouTubeConfig youTubeConfig;
+
+    @Autowired
+    RadioConfig radioConfig;
+
+    private YouTube youTube;
+
+    public Video getYouTubeVideoById(String videoId) {
+        try {
+            VideoListResponse response;
+            youTube = getYouTube();
+
+            HashMap<String, String> parameters = new HashMap<>();
+            parameters.put("part", "id,snippet,contentDetails");
+            parameters.put("id", videoId);
+
+            YouTube.Videos.List videosListByIdRequest = youTube.videos().list(parameters.get("part").toString());
+            videosListByIdRequest.setKey(youTubeConfig.getApiKey());
+            if (parameters.containsKey("id") && parameters.get("id") != "") {
```

Comment:
We can not compare string with !=, please change to equals, or use StringUtils.isEmpty(parameters.get("id"))

Reply:
I fixed it